packages/solid/src/components/image-cropper/examples/basic.tsx

```diff
@@ -0,0 +1,26 @@
+import { For } from 'solid-js'
+import { ImageCropper } from '../'
+
+export const Basic = () => {
+  const handlePositions = [
+    'top-left',
+    'top',
+    'top-right',
+    'right',
+    'bottom-right',
+    'bottom',
+    'bottom-left',
+    'left',
+  ] as const
+
```

Comment:
In the next zag version, we export the `imageCropper.handles` to replace this array